packages/eslint-config-react/package.json

```diff
@@ -10,10 +10,14 @@
     "Robert Rossmann <rr.rossmann@me.com>",
     "Danny Kijkov <daniel.kijkov@strv.com>"
   ],
+  "peerDependencies": {
```

Comment:
Could you, pretty please, run `npx fixpack packages/eslint-config-react/package.json` to make sure the _package.json_ file is formatted and its keys sorted uniformly across all packages here? 🎨 🙏